packages/graphql-transformers-e2e-tests/src/__tests__/NewConnectionTransformer.e2e.test.ts

```diff
@@ -0,0 +1,367 @@
+            childName
+        }
+    }`, {})
+    expect(createParent1.data.createParent.id).toBeDefined()
+    expect(createParent1.data.createParent.childID).toEqual(createChild.data.createChild.id)
+    expect(createParent1.data.createParent.childName).toEqual(createChild.data.createChild.name)
+
+    const queryChild = await GRAPHQL_CLIENT.query(`query {
+        getChild(id: "${createChild.data.createChild.id}", name: "${createChild.data.createChild.name}") {
+            id
+            parents {
+                items {
+                    id
+                    childID
+                    childName
+                }
+            }
+        }
+    }`, {})
+    expect(queryChild.data.getChild).toBeDefined()
+    const items = queryChild.data.getChild.parents.items
+    expect(items.length).toEqual(1)
+    expect(items[0].id).toEqual(createParent1.data.createParent.id)
+    expect(items[0].childID).toEqual(createParent1.data.createParent.childID)
+    expect(items[0].childName).toEqual(createParent1.data.createParent.childName)
+})
+
+test('Test PostModel.authors query with composite sortkey', async () => {
+    const createUser = await GRAPHQL_CLIENT.query(`mutation {
+        createUser(input: { id: "123" name: "Bob", surname: "Rob" }) {
+          id
+          name
+          surname
+        }
+      }`, {})
+
+      expect(createUser.data.createUser.id).toBeDefined()
+      expect(createUser.data.createUser.name).toEqual('Bob')
+      expect(createUser.data.createUser.surname).toEqual('Rob')
+    const createPostModel = await GRAPHQL_CLIENT.query(`mutation {
+        createPostModel(input: { authorID: "${createUser.data.createUser.id}",
+                                 authorName: "${createUser.data.createUser.name}",
+                                 authorSurname: "${createUser.data.createUser.surname}",
+                                 postContents: "potato"
+                                }
+                        )
+        {
+          id
+          authorID
+          authorName
+          authorSurname
+          postContents
+        }
+      }`, {})
+    expect(createPostModel.data.createPostModel.id).toBeDefined()
+    expect(createPostModel.data.createPostModel.authorID).toEqual(createUser.data.createUser.id)
+    expect(createPostModel.data.createPostModel.authorName).toEqual(createUser.data.createUser.name)
+    expect(createPostModel.data.createPostModel.authorSurname).toEqual(createUser.data.createUser.surname)
+    const queryPostModel = await GRAPHQL_CLIENT.query(`query {
+        getPostModel(id: "${createPostModel.data.createPostModel.id}") {
+            id
+            authors {
+                items {
+                    id
+                    name
+                    surname
+                }
+            }
+        }
+    }`, {})
+    expect(queryPostModel.data.getPostModel).toBeDefined()
+    const items = queryPostModel.data.getPostModel.authors.items
+    expect(items.length).toEqual(1)
```

Comment:
If I have more than one value with the same id, name, surname combo will they appear here as expected?

Reply:
Yes, added a test for this.